.github/workflows/test-k8s-service-buffer.yml

```diff
@@ -16,6 +16,7 @@ env:
 
   KUBERNETES_NAMESPACE: buffer
   KUBERNETES_ROLLOUT_WAIT: 120s
+  REMOVE_RESOURCES_LIMITS: true
```

Comment:
Nakoniec sa mi podarilo zistit, preco to v CI nefunguje.

Github Actions totiz bezia na 2 CPU, a ako mame `request: cpu: 200m` pre kazdy pod, 
tak pri rollout uz nie je miesto pre nove pods.

Preto musim v CI vypnut resources limits.